### ISSUE TITLE
MATLAB package should be in the exclude list

### ISSUE THREAD

**Issue by GravityAssisted** [rened/SystemImageBuilder.jl#1]
Hi,

I tried building the image and noticed that the MATLAB.jl package by @simonster stops working. In particular the `MSession()` and `eval_string` functions are not defined.

``` julia
julia> MATLAB.MSession()
ERROR: MSession not defined

julia> MATLAB.eval_string()
ERROR: eval_string not defined

```

I have put this package in my exclude list for now. 

BTW: The SystemImageBuilder package is awesome, I see a huge improvement in my workflow ! Thank you :+1:

**Comment by rened** [comment]
Hi,

thanks a lot for the info! 

@simonster it looks like the calls to `load_libmx()`and `load_libmat()` could be placed within a `__init__()` to make this work?

**Comment by GravityAssisted** [comment]
That seems to be the true @rened and maybe we need to put `get_paths()` in `__init__()` as well... ?

**Comment by rened** [comment]
Hm, possibly, yes! Unfortunately I can't test this at the moment, my Julia/Matlab setup is somehow borked - could you try that out, please?

**Comment by GravityAssisted** [comment]
Sure,  I will try out and get back to you. It may have to wait until morning as its late in this part of the world :)  (California)

**Comment by rened** [comment]
I tried to put these functions in `__init()__`, but could not get it to work. I am putting `MATLAB` on the exclude list, we can enable it again later. Thanks again for reporting thing this!
Closed by d7694a51b1f26876ffb562059f9d567075d635ef.
